### PR TITLE
Split optimal savings

### PR DESCRIPTION
**Pull request by jstac**
Split optimal savings lecture in two, add vmap version, numpy comparison.

### CODE REVIEW

**Comment by jstac** [comment]
@mmcky Would you mind to see if you can tidy this up?

Once it's fixed and merged, could you please refresh the live site.

**Comment by mmcky** [comment]
@jstac when you say `tidy-up` do you mean `proof-read` and look out for any style regressions?

**Comment by mmcky** [comment]
@jstac [07bb3d3](https://github.com/QuantEcon/lecture-jax/pull/151/commits/07bb3d393a7aef67139b98a1b022f95f15410cb4) just makes minor modifications in `linking` and adding `figure` numbers etc.

File: lectures/opt_savings_2.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ def create_consumption_model(R=1.01,                    # Gross interest rate
 Here's the right hand side of the Bellman equation:
 
 ```{code-cell} ipython3
+:tags: [hide-input]
```
Comment (mmcky) [inline lectures/opt_savings_2.md:115]: @jstac do you want this to be hidden or visible?

Comment (jstac) [reply lectures/opt_savings_2.md:115]: Good catch!  All code should be visible in both of the two lectures.

**Comment by mmcky** [comment]
@jstac this is building and looking good 👍

**Comment by mmcky** [comment]
> Once it's fixed and merged, could you please refresh the live site.

I will merge this now and update the live site.

**Comment by jstac** [comment]
Thanks @mmcky , please let me know when the new version is live.

**Comment by mmcky** [comment]
I'll trigger a build as soon as the cache has finished building 👍

**Comment by mmcky** [comment]
@jstac new version is now live.

**Pull request merged**